examples/messages/src/App.tsx

```diff
@@ -21,15 +22,15 @@ function App() {
         channel: "TEST",
         account: account,
         fileObject: file,
-        storageEngine: StorageEngine.STORAGE,
+        storageEngine: ItemType.storage,
         APIServer: DEFAULT_API_V2,
       });
 
       console.log(confirmation)
     })();
   }, []);
   return (
-      <div>
+    <div>
     </div>
   );
```

Comment:
What is the point of using React to simply run a side effect with no update to the dom? :raised_eyebrow: